src/OpenTK/Platform/SDL2/Sdl2Keyboard.cs

```diff
@@ -38,28 +38,12 @@ public Sdl2Keyboard()
             state.IsConnected = true;
         }
 
-        // Unfortunately, SDL does not report KeyDown events
-        // when a modifier (e.g. shift, alt, etc) is first pressed.
-        // It reports a keydown+keyup event pair when the modifier
-        // is *released* - which means that we cannot use modifiers
-        // for regular input (e.g. press control to fire a weapon.)
-        // For that reason, we should also poll the keyboard directly
-        // as necessary.
-        // Fixme: this does not appear to work as expected.
```

Comment:
So this was dead code that was never called but while here can we check if KeyDown events for Ctrl, Alt, NumLock etc work or not. If not we should raise a github issue.